app/code/Magento/Cms/Model/Wysiwyg/Images/Storage.php

```diff
@@ -416,7 +416,7 @@ public function createDirectory($name, $path)
     {
         if (!preg_match(self::DIRECTORY_NAME_REGEXP, $name)) {
             throw new \Magento\Framework\Exception\LocalizedException(
-                __('Please rename the folder using only letters, numbers, underscores and dashes.')
+                __('Please rename the folder using only Latin letters, numbers, underscores and dashes.')
```

Comment:
Could you please also add the new translation string to `Magento_Cms/i18n/en_US.csv`?